### DISCUSSION TITLE
FAQ

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Gordian#74] [General Discussion]
This thread is intended to be the start of a FAQ, to eventually be made into a repo file.

**Q:** What is the Gordian System?

**A:** It's a full node linked with a mobile wallet.

**Q:** Why use the Gordian System?

**A:** There are a few advantages to this setup:
* *Security.* A mobile wallet allows you to keep your private keys away from devices directly connected to the internet.
* *Autonomy.* Maintaining your own full node ensures that you can't be censored, tracked, or otherwise taken advantage of by someone else. It ensures your self-sovereignty.

**Q:** What OSes can the Gordian System run on?

**A:** The Gordian server runs on MacOS, though the Bitcoin Standup technology that's at its foundation can also be run on UNIX machines, with the Linodes running Debian being particularly well tested.

The Gordian Wallets runs under iOS.

_If you have any Gordian related questions that you'd like answered, please post them here._

**Comment by henkvancann** [comment]
First version https://github.com/henkvancann/GordianServer-macOS/blob/master/Docs/Q-and-A.md
